hackathon_site/dashboard/frontend/src/components/inventory/Item/Item.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import styles from "./Item.module.scss";
+import Card from "@material-ui/core/Card";
+import CardMedia from "@material-ui/core/CardMedia";
+import FiberManualRecordIcon from "@material-ui/icons/FiberManualRecord";
+import Grid from "@material-ui/core/Grid";
+import Container from "@material-ui/core/Container";
+import Paper from "@material-ui/core/Paper";
+
+const colorCircle = ({ limit, currentStock }) => {
```

Comment:
Change the component name to ColorCircle. I also don't think currentStock is relevant to render a circle. I think we will set limit to null if there is no limit. Also instead of if statements, I would use a switch statement just cause it's a little cleaner.